sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/IntervalUtils.scala

```diff
@@ -391,6 +392,43 @@ object IntervalUtils {
     fromDoubles(interval.months / num, interval.days / num, interval.microseconds / num)
   }
 
+  def toSqlStandardString(interval: CalendarInterval): String = {
```

Comment:
we can move `CalendarInterval.toString` to here with name `toMultiUnitsString`. `CalendarInterval.toString` should be as simple as `CalendarInterval(months = ..., days = ..., microseconds = ...)`

Reply:
We can do it in a followup

Reply:
Yea sounds great